examples/performance-subscribe/src/PerformanceTestApp.cpp

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2022-2024 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "PerformanceTestApp.h"
+#include "sdk/IPubSubClient.h"
+#include "sdk/Logger.h"
+#include "sdk/vdb/IVehicleDataBrokerClient.h"
+
+#include <csignal>
+#include <fstream>
+#include <iostream>
+#include <nlohmann/json.hpp>
+#include <utility>
+
+namespace example {
+
+namespace {
+std::string getValueRepresentation(const velocitas::DataPointValue& value) {
+    if (!value.isValid()) {
+        return toString(value.getFailure());
+    }
+    return value.getValueAsString();
+}
+
+std::ostream& operator<<(std::ostream&                                        ostr,
+                         const std::chrono::high_resolution_clock::time_point timestamp) {
+    std::time_t time = std::chrono::high_resolution_clock::to_time_t(timestamp);
+    uint64_t    useconds =
+        std::chrono::duration_cast<std::chrono::microseconds>(timestamp.time_since_epoch())
+            .count() %
+        std::micro::den;
```

Comment:
just my personal opinion but this formatting looks really ugly, maybe we should think about changing some settings?

Reply:
Yes, this std::chrono stuff is somewhat "bulky". 
What do you mean by "changing some settings"?

Reply:
I meant our formatting settings... but as mentioned this is just my personal opinion

Reply:
To be done later